README.md

```diff
@@ -32,7 +32,7 @@ Or, you can find a config file for etherpad at this path `/var/www/etherpad_mypa
 
 #### Multi-users support
 
-Supported, **without LDAP nor SSO**.
+Supported, with LDAP.
```

Comment:
What about writing "with or without SSO"...?
That also raises a question for the install parameter (`Voulez-vous utiliser ldap avec mypads ? ` in French). I think it would be better to stick to "SSO" rather than "LDAP" terminology (concept vs implementation).

Reply:
I don't think that's only a matter of terminology, LDAP and SSO are not obviously linked.
And for this app, I don't think it would be any HTTP Auth support (not try yet though)

Reply:
In understand what you mean: SSO covers 2 concepts (unique directory AND single authentication).
Yet, using "LDAP" doesn't feel totally right for me... Should we propose a more abstract term, such as "YunoHost directory"...?

Reply:
I'm not really for the idea of replacing technical words by cool names to appears less difficult.

Ldap is ldap, I can understand that ldap isn't a simple concept (actually, I can't find a simple explanation of what is ldap...)
But, I rather prefer a documentation to link to the term ldap than another word. But actually we haven't any documentation on this matter yet.
I think I glossary would be great to explain that kind of concepts.

Reply:
Yes, I think a glossary could be a good idea. I tend to agree with Maniack, LDAP is LDAP, we won't replace LDAP name by another. It's a non-sense. The other name won't cover anyway the global meaning of LDAP. Moreover, I think, as we use to write LDAP, the Lambda user will have to understand the concept and that is good. Yunohost is to make easier adminsys, it's not to create words to make it more difficult. :-) ((It's my point of view, only))